app/assets/javascripts/components/add_to_playlist.js.jsx

```diff
@@ -29,12 +29,12 @@ class AddToPlaylist extends React.Component {
     $.ajax({
       url: "/api/v1/playlists/" + playlistId + ".json",
       method: "PATCH",
-      data: {user_id: userId, playlist_id: playlistId, track_uri: trackUri },
+      data: {user_id: userId, playlist_id: playlistId, track_uri: trackUri},
       success: () => {
         $("#add-to-playlist").hide();
         $("#playlists-listing").hide();
-        $("#added-message").slideDown( () => {
-          setTimeout( () => {
+        $("#added-message").slideDown(() => {
+          setTimeout(() => {
```

Comment:
I think these would be a bit nicer if you have the time to spend to have react to do these instead of jquery